types/experiments.ts

```diff
@@ -0,0 +1,72 @@
+/** All fields are publically accessible via the API and available to clients */
+export interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  public_name: string;
```

Comment:
@jaredlockhart is it OK to change this or do we need to keep this `name`?

Reply:
@k88hudson Yeah we can call it whatever we want here and just rename it when we serialize, though it might be a bit confusing when you look at other parts of the Experimenter API or other parts of the UI and it's called `name` and not `public_name`, but that might also be true of other things when comparing the rapid API to the non rapid API.

But maybe one progressive way to look at it is figuring out how we want the rapid stuff to look in isolation without being fettered by past designs and just doing what makes sense for this effort?